src/System.ComponentModel.Composition/tests/System/Integration/AdaptingCollectionTests.cs

```diff
@@ -343,6 +343,7 @@ public class DynamicExports
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]
```

Comment:
Why is this the "fix"?

Reply:
My mistake. I shouldn't have said this is the fix. 
I'm pushing up a skip commit meanwhile I'm investigating the reason for failure.

Reply:
I believe Dan and I have a difference of opinion here, but if it were me, I'd want to see a comment next to this with the corresponding issue number, so that a) someone reading the code can find more info on why this is skipped, and so that b) someone looking at the issue can do a code search for the issue number to find all of the relevant tests disabled against it.

Reply:
The issues 25598 to 25602 point to 5 failing tests on uap related to newly ported System.ComponentModel.Composition tests, with each issue specifying one failing test.

I'll push up a single commit skipping all 6 test, and linking them with issue 25498 as well as the aforementioned 6 newly opened issues.

Reply:
Actually, here I would use `[ActiveIssue(25498, TargetFrameworkMonikers.Uap)]` instead. As the docs say the purpose of active issue is to disable a test related to a bug that needs to be investigated/resolved, SkipOnTargetFramework is more of, "we have a difference in behavior when testing on this platform, so we skip it".

Reply:
Thanks @stephentoub and @safern . I updated the commit as suggested.

Reply:
> I believe Dan and I have a difference of opinion here

Actually I'm completely in favor of issue numbers and history in test comments. 

For product code I believe we have common ground that we should use discretion to avoid a rats nest of bug numbers. (I've seen codebases that ended up that way and it has bad consequences). Of course, if code can be satisfactorily explained in a comment without reference to a bug number, then there is no need for one.

Reply:
> Actually I'm completely in favor of issue numbers and history in test comments.

Ah, ok, good then :smile: